src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Exceptions/AnonymizationProcessFailedException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Microsoft.Health.Fhir.Anonymizer.Core.Exceptions
+{
+    // Process runtime exception.
+    public class AnonymizationProcessFailedException : Exception
```

Comment:
:yellow_circle: Suggestion
The two kinds of exceptions are named `Anonymizer...` and `Anonymization...`. Maybe we can use a same prefix to make them more consistent.

Reply:
Thanks, fixed!